charts/jenkins/README.md

```diff
@@ -341,12 +341,24 @@ If the storage class is set to null or left undefined (`""`), the default provis
 
 #### Additional Secrets
 
-Additional secrets can be mounted into the Jenkins controller through the chart or created using `controller.additionalSecrets`. A common use case might be identity provider credentials if using an external LDAP or OIDC-based identity provider.
+Additional secrets and Additional External Secrets,
+can be mounted into the Jenkins controller through the chart or created using `controller.additionalSecrets` or `controller.additionalExistingSecrets`.  
+A common use case might be identity provider credentials if using an external LDAP or OIDC-based identity provider.
 The secret may then be referenced in JCasC configuration (see [JCasC configuration](#configuration-as-code)).
 
 `values.yaml` controller section, referencing mounted secrets:
 ```yaml
 controller:
+  # 'name' and 'keyName' must be lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-',
+  # and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc')
+  additionalExistingSecrets:
+    - name: secret-credentials
+      keyName: github-username
+    - name: secret-credentials
+      keyName: github-password
+   - name: secret-credentials
+      keyName: token
```

Comment:
```suggestion
   - name: secret-credentials
     keyName: token
```

Reply:
Without looking at the configScripts example I would not know that `name` and `keyName` are concatenated                   `secret: ${secret-credentials-token}`.
I thinks that's worth documenting here.

Reply:
readme and comment inside values file updated